nicegui/outbox.py

```diff
@@ -23,35 +23,46 @@ def __init__(self, client: Client) -> None:
         self.updates: Dict[ElementId, Optional[Element]] = {}
         self.messages: Deque[Message] = deque()
         self._should_stop = False
+        self._enqueue_event = None
```

Comment:
Why did you chose lazy initialization? Can't we simply create the event here and simply set it later in the enqueue methods? This would make `_set_enqueue_event` kind of obsolete.

Reply:
That was exactly what I did at first and it worked flawlessly with Python 3.12. Testing with 3.8, however, caused the following exception when the event was accessed: `RuntimeError: Task got Future <Future pending> attached to a different loop`. This is because prior to Python 3.10, events attach themselves to the current event loop during initialization. In this case, it appears that the event loop Outbox is created on is different from the one executing its loop method. I switched to the lazy initialization approach to maintain compatibility with Python 3.8.

Reply:
Oh sorry, I just noticed your explanation in your original post. Sorry! 🫣